Description: fix missing interior of rectangles (regression in 9.54.0)
 This was caused by the commit bdf7cf8b6c.
 If overprint was true,
 in the sections of the 8 and 16 bit do_mark_fill_rectangle_ko_simple
 that copy the 'dst' pixel to the plane buffers (dst_ptr),
 the colors were copied, but not the alpha.

 For the example file,
 the fill_stroke had the fill end up transparent and the stroke be visible,
 since stroke_overprint was false.
Origin: upstream, https://git.ghostscript.com/?p=ghostpdl.git;h=c6ce09a703848
Author: Ray Johnston <ray.johnston@artifex.com>
Bug: https://bugs.ghostscript.com/show_bug.cgi?id=703848
Last-Update: 2021-18-07
---
This patch header follows DEP-3: https://dep.debian.net/deps/dep3/
--- a/base/gdevp14.c
+++ b/base/gdevp14.c
@@ -7726,6 +7726,7 @@
                             dst_ptr[k * planestride] = dst2[k];
                         }
                     }
+                    dst_ptr[num_comp * planestride] = dst[num_comp];    /* alpha */
                 }
             } else {
                 if (overprint) {
@@ -7942,6 +7943,7 @@
                             dst_ptr[k * planestride] = dst2[k];
                         }
                     }
+                    dst_ptr[num_comp * planestride] = dst[num_comp];    /* alpha */
                 }
             } else {
                 if (overprint) {
